src/plugins/timelion/public/components/timelion_deprecation.tsx

```diff
@@ -19,7 +19,7 @@ export const TimelionDeprecation = ({ links }: DocLinksStart) => {
         title={
           <FormattedMessage
             id="timelion.deprecation.message"
-            defaultMessage="Deprecated since 7.0, the Timelion app will be removed in 8.0. To continue using your Timelion worksheets, {timeLionDeprecationLink}."
+            defaultMessage="Deprecated since 7.0, the Timelion app will be removed in the latest 7.x minor. To continue using your Timelion worksheets, {timeLionDeprecationLink}."
```

Comment:
```suggestion
            defaultMessage="Deprecated since 7.0, the Timelion app will be removed in the last 7.x minor version. To continue using your Timelion worksheets, {timeLionDeprecationLink}."
```